### ISSUE TITLE
Model size

### ISSUE THREAD

**Issue by leyuan-sun** [ArashJavan/DeepLIO#21]
What params inside deeplio, compared with the size of trained model between deeplio and lidar-feat-net, the former one is about 400+MB, and the latter one is only 20+MB, so I am wonder why the size of deeplio model is much bigger than lidar-feat-net model.

deeplio:
  dropout: 0.25
  pretrained: false
  model-path: ""
  lidar-feat-net:
    name: "lidar-feat-pointseg"
    pretrained: false
    model-path: ""
    requires-grad: true
